Niyonizeye/AnnounceIT
# User should be able for Create an Account

### What does this PR do
- Allow User to Create an Account
### Description of task to be completed
- Added Html pages
- Added Style
### How should this be manually tested
- Clone git repository
- https://github.com/Niyonizeye/AnnounceIT.git
- Open index.html in the browser
- See on Right of home page
- Click on Signup 
### Any background context you want to provide?
- N/A

### What are relevant pivotal tracker stories ?
- https://www.pivotaltracker.com/story/show/170744347

### screenshots (if appropriate)
![signup](https://user-images.githubusercontent.com/45454232/73019664-b555be80-3e2c-11ea-8d4f-08524f6feaf6.PNG)



### Questions
- N/A
